test/k8sT/Services.go

```diff
@@ -301,14 +339,18 @@ var _ = Describe("K8sServicesTest", func() {
 			// teardown if any step fails.
 			_ = kubectl.Delete(demoYAML)
 			_ = kubectl.Delete(echoSVCYAML)
+			if dualStackSupportEnabled {
+				_ = kubectl.Delete(demoYAMLDualStack)
+				_ = kubectl.Delete(echoSVCYAMLDualStack)
+			}
 		})
 
 		SkipItIf(helpers.RunsWithoutKubeProxy, "Checks service on same node", func() {
-			clusterIP, _, err := kubectl.GetServiceHostPort(helpers.DefaultNamespace, serviceName)
-			Expect(err).Should(BeNil(), "Cannot get service %s", serviceName)
-			Expect(govalidator.IsIP(clusterIP)).Should(BeTrue(), "ClusterIP is not an IP")
+			serviceNames := []string{serviceName}
+			if dualStackSupportEnabled {
```

Comment:
This looks odd, we can't "know" the order the variable will be initialized so it would be better executing `helpers.DualStackSupported()` directly here.